error-handler.js

```diff
@@ -269,6 +277,9 @@ createHandler = function createHandler(options) {
       return resumeOrClose(status);
     }
 
+    // Always set a response status.
```

Comment:
> Also please consider that the 500 conformation in the fall-through can hide crucial status information

That's the idea. We want to hide error nuances for security reasons. Doing so is a good default for production web services, but we should probably expose a way to customize exceptions for applications that depend on additional error information.

The more you reveal about the server, less less surface area you expose to clients with bad intentions. But we still need to be friendly to developers attempting to debug system faults.

Ideally, the real error gets logged for the developers maintaining the system, but a generic 500 error is returned so those looking at the service from the outside can't gain any insights about whether or not their exploit attempts are on the right track.